<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>706</width>
    <height>951</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="title">
       <string>GroupBox</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QListWidget" name="listWidget">
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>150</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="runButtons">
         <property name="maximumSize">
          <size>
           <width>150</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Run</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget" native="true">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QGroupBox" name="originalGroup">
         <property name="title">
          <string>Original/Grey</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>337</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="originalImage">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>120</height>
             </size>
            </property>
            <property name="text">
             <string>Image</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="grayImage">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>120</height>
             </size>
            </property>
            <property name="text">
             <string>Image</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="brightnessThresholdGroup">
         <property name="title">
          <string>Brightness Threshold</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="3" rowspan="2">
           <widget class="QLabel" name="brightnessThresholdImage">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>120</height>
             </size>
            </property>
            <property name="text">
             <string>Image</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>291</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Threshold</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QDoubleSpinBox" name="brightnessSpinBox">
            <property name="decimals">
             <number>0</number>
            </property>
            <property name="maximum">
             <double>255.000000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="colorThresholdGroup">
         <property name="title">
          <string>Color Threshold</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="6" column="2">
           <widget class="QDoubleSpinBox" name="colorDliateSpin">
            <property name="decimals">
             <number>0</number>
            </property>
           </widget>
          </item>
          <item row="4" column="2">
           <widget class="QDoubleSpinBox" name="rbChannelThresholdSpinBox">
            <property name="decimals">
             <number>0</number>
            </property>
            <property name="maximum">
             <double>255.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="0" column="1" colspan="4">
           <widget class="QRadioButton" name="hsvRadio">
            <property name="text">
             <string>HSV</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QDoubleSpinBox" name="hsvRedHueMinSpinBox">
            <property name="decimals">
             <number>0</number>
            </property>
            <property name="minimum">
             <double>90.000000000000000</double>
            </property>
            <property name="maximum">
             <double>180.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="1" column="3">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>to</string>
            </property>
           </widget>
          </item>
          <item row="6" column="1">
           <widget class="QCheckBox" name="colorDliateCheck">
            <property name="text">
             <string>Dliate</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Red hue from</string>
            </property>
           </widget>
          </item>
          <item row="2" column="4">
           <widget class="QDoubleSpinBox" name="hsvBlueHueMaxSpinBox">
            <property name="decimals">
             <number>0</number>
            </property>
            <property name="maximum">
             <double>180.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Threshold</string>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QDoubleSpinBox" name="hsvBlueHueMinSpinBox">
            <property name="decimals">
             <number>0</number>
            </property>
            <property name="maximum">
             <double>180.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="0" column="6" rowspan="6">
           <widget class="QLabel" name="colorThresholdImage">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>120</height>
             </size>
            </property>
            <property name="text">
             <string>Image</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="1" column="4">
           <widget class="QDoubleSpinBox" name="hsvRedHueMaxSpinBox">
            <property name="decimals">
             <number>0</number>
            </property>
            <property name="maximum">
             <double>90.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="2" column="3">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>to</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Blue hue from</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1" colspan="4">
           <widget class="QRadioButton" name="rbChannelRadio">
            <property name="text">
             <string>R-B Channels</string>
            </property>
           </widget>
          </item>
          <item row="5" column="1" colspan="4">
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Minimum</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>10</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="2" column="5">
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>337</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="contourGroup">
         <property name="title">
          <string>Contour</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <spacer name="horizontalSpacer_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>337</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="contourImage">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>240</height>
             </size>
            </property>
            <property name="text">
             <string>Image</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
